R/crmPack-package.R

```diff
@@ -33,7 +33,7 @@
 #' @importFrom graphics plot hist legend lines matlines matplot
 #' @importFrom stats binomial coef cov2cor gaussian glm lm median model.matrix
 #' optim pgamma plogis pnorm qgamma qlogis qnorm quantile rbinom rgamma approxfun
-#' rnorm runif uniroot var vcov step mad pbeta dbeta dgamma setNames
+#' rnorm runif uniroot var vcov step mad pbeta dbeta dgamma setNames sd
```

Comment:
instead of this (I thought I mentioned this before but not sure) would be better to just import a single function and always use `stats::fun` for any `fun`.

Reply:
Yap, you mentioned about this - but I was not sure which option was the correct one, and I chose wrong one :( sorry :).